### DISCUSSION TITLE
homebrew won't install mosquitto@1.6.12

### DISCUSSION THREAD

**Discussion by ajoslin103** [Homebrew/discussions#244] [Casks]
homebrew won't install mosquitto@1.6.12  

how can I get brew to install mosquitto@1.6.12

**Comment by gromgit** [comment]
You'd have to extract that formula version to your own tap, then install it from there, e.g.:
```
brew tap-new ajoslin103/tap
brew extract --version 1.6.12 mosquitto ajoslin103/tap
brew install ajoslin103/tap/mosquitto@1.6.12
```

**Comment by carlocab** [comment]
There is no formula for mosquitto@1.6.12. If you need this version, one way to install it using brew is to:
1. [create a tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap)
2. extract `mosquitto@1.6.12` to your tap using `brew extract --version 1.6.12 mosquito <your tap name here>`
3. `brew install mosquitto@1.6.12`

**Comment by ajoslin103** [comment]
I thought it would be something like that -- I forked the whole homebrew-core just to update the mosquitto formula and then tapped it but that didn't work -- so now I have to untap [and delete a POC] and do it this way :)

which worked!  thanks!

**Comment by ajoslin103** [comment]
not that this fixed my problem with connecting websockets via port :8000 

something changed this week...  

Not using Big Sur yet...

**Reply by carlocab** [reply]
I don't use mosquitto, so I can't help you there. You'll probably need to ask for help upstream.

**Comment by ajoslin103** [comment]
yar!  

I have, thanks
